server/sonar-server/src/main/java/org/sonar/server/notification/DefaultNotificationManager.java

```diff
@@ -112,54 +106,42 @@ void logDeserializationIssue() {
   }
 
   public long count() {
-    return notificationQueueDao.count();
+    return dbClient.notificationQueueDao().count();
   }
 
   /**
    * {@inheritDoc}
    */
   @Override
-  public Multimap<String, NotificationChannel> findSubscribedRecipientsForDispatcher(NotificationDispatcher dispatcher,
-    @Nullable String projectUuid) {
+  public Multimap<String, NotificationChannel> findSubscribedRecipientsForDispatcher(NotificationDispatcher dispatcher, String projectUuid) {
+    checkArgument(projectUuid != null, "ProjectUUID is mandatory");
```

Comment:
We usually use `Objects.requireNonNull` to check nulls, which throws NPE rather than throwing IllegalArgumentException